src/components/SelectedTapHours.js

```diff
@@ -14,7 +14,7 @@ const SelectedTapHours = ({ infoIsExpanded, selectedPlace }) => {
 
   useEffect(() => {
     // update hoursList, currentOrgHours, and isOpen when selectedPlace changes
-    if (selectedPlace.hours !== undefined) {
+    if (selectedPlace.hours && selectedPlace.hours.length === 7) {
```

Comment:
This will ensure that the hour related data is null if we don't have all 7 entries in the dataset. "Closed" will be displayed without a dropdown.